README.md

```diff
@@ -66,236 +68,10 @@ npm run develop
-| E2E_SUPPLIER_PASSWORD | Supplier user password used for acceptance testing | |
-| LOCATIONS_BATCH_SIZE | Maximum number of location IDs to send in one request when requesting moves for all locations | 40 |
-| FEATURE_FLAG_PERSON_ESCORT_RECORD | Set to `true` to enable display of the Person Escort Record feature | |
-| FRAMEWORKS_VERSION | Current [Book a secure move frameworks](https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks/releases) version that the frontend will use to create new Person Escort Records or any other frameworks being used | latest supported version (see `@hmpps-book-secure-move-frameworks` in package.json) |
-
-### Development specific
-
-The following environment variables can be set to help development.
-
-| Name | Description | Default |
-|:-----|:------------|:--------|
-| BYPASS_SSO | Set to `true` to bypass authentication | |
-| USER_PERMISSIONS | Comma delimited string of available permissions (required if bypassing auth) | |
-| USER_LOCATIONS | Comma delimited string of available locations (required if bypassing auth) | |
-
-## Components
-
-See the [components readme](./common/components/README.md) for more detail on how to structure app level components.
-
-## Book a secure move frameworks
-
-The frontend uses the [Book a secure move frameworks](https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks) repo to build and maintain
-certain features like the Person Escort Record.
-
-The frontend maintains a list of versions of the framework that are supported in the package.json manifest under a scope of `@hmpps-book-secure-move-frameworks`. Each version is then maintained as a folder underneath that scope, for example `@hmpps-book-secure-move-frameworks/0.1.0`.
-
-The frontend needs to maintain separate versions so that it can render any previous records created using the framework or allow users to continue
-to edit existing records that have been created using specific versions.
-
-### To support a new version
-
-- Add a new item under the framework scope to the package manifest:
+## Licence
 
-  ```
-  "@hmpps-book-secure-move-frameworks/{semanticVersion}": "github:ministryofjustice/hmpps-book-secure-move-frameworks#v{semanticVersion}"
-  ```
+Unless stated otherwise, the codebase is released under the MIT License. This covers both the codebase and any sample code in the documentation. The documentation is © Crown copyright and available under the terms of the Open Government 3.0 licence.
 
-### To override the current version
+## Contribution guidelines
 
-The current version is used when creating new framework resources. It can be overridden using an environment variable (`FRAMEWORKS_VERSION`),
-for example when adding support for a new version but we want to still create records using an older version.
+If you want to help us improve Book a Secure Move, view our [contribution guidelines](./CONTRIBUTING.md)
```

Comment:
We've (/I've) tried to avoid capitalising the S and M from the service name to avoid people turning it into a acronym :) Would be good to remain consistent with the other uses in this repo and use `Book a secure move`.